lms/templates/instructor/instructor_dashboard_2/add_coupon_modal.html

```diff
@@ -2,6 +2,7 @@
 from django.utils.translation import ugettext as _
 from django.core.urlresolvers import reverse
 %>
+<%page expression_filter="h"/>
 <%page args="section_data"/>
```

Comment:
We learned that there can only be one page expression on the page.  Please combine these. :)  I need to update the docs.